.github/workflows/test.yml

```diff
@@ -0,0 +1,53 @@
+name: Run tests
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+jobs:
+    test:
+      runs-on: ubuntu-latest
+      strategy:
+        matrix:
+          python: [2.7, 3.6, 3.7, 3.8, 3.9]
+      steps:
+        - uses: actions/checkout@v2
+        - name: Setup Python
+          uses: actions/setup-python@v2
+          with:
+            python-version: ${{ matrix.python }}
+        - name: Install Tox and any other packages
+          run: |
+            python -m pip install --upgrade pip
+            pip install tox tox-gh-actions
+        - name: Run Tox
+          # Run tox using the version of Python in `PATH`
+          run: tox -epy
+    flake8:
```

Comment:
I think you probably know this, but just to be sure - I believe this sets up a separate container to run flake8.  This may not be an issue, but it is likely to make the tests run a bit slower.  I believe on the ones I changed, I just made flake8 a step, which would also run it for each of the pythons listed above.

Reply:
I did this on purpose indeed, since it seems slower and unnecessary to me to repeat the flake8 step for every version?

Reply:
Also fair